code/_onclick/hud/action_button.dm

```diff
@@ -169,10 +169,12 @@
 	.["bg_state_active"] = "template_active"
 
 /atom/movable/screen/movable/action_button/proc/set_to_keybind(mob/user)
-	var/keybind_to_set_to = uppertext(input(user, "What keybind do you want to set this action button to?") as text)
+	var/keybind_to_set_to = input(user, "What keybind do you want to set this action button to?") as text
 	if(keybind_to_set_to)
 		if(linked_keybind)
 			clean_up_keybinds(user)
+		if(length_char(keybind_to_set_to) <= 3)
```

Comment:
This random length check addition should at least have a comment explaining what it's doing and why it's necessary

Reply:
Wait a moment